src/fio/FioEngine.ts

```diff
@@ -168,7 +168,10 @@ export class FioEngine extends CurrencyEngine<FioTools, SafeFioWalletInfo> {
   ) {
     super(env, tools, walletInfo, opts)
     this.fetchCors = getFetchCors(env.io)
-    this.tpid = tpid
+    // Removed tpid as a temporary fix for connecting wallets using bundled tx's
+    // until FIO fixes
+    // their chain
+    // this.tpid = tpid
```

Comment:
The network bug only affects zero balance wallets so we should still set it in the constructor but make it undefined in the balance loop if necessary